Header header
geometry_msgs/Pose pose
std_msgs/Float64 alpha_red
std_msgs/Float64 alpha_green
std_msgs/Float64 alpha_cyan
std_msgs/Float64 alpha_purple
